### PR TITLE
chore: enabled v1 verification in filter and url_params test files

### PR DESCRIPTION
**Pull request by adam-stasiak**
### SUMMARY

I added aliases for v1 charts verification.
cc @villebro 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts [hunk 1]
```diff
@@ -93,20 +90,30 @@ describe('Dashboard filter', () => {
...
+              col: 'region',
+              op: '==',
+              val: 'South Asia',
+            });
+          } else {
+            responseBody.result.forEach((element: any) => {
+              expect(element.applied_filters[0]).deep.eq({ column: 'region' });
+              expect(element.data[0]).to.have.property('region', 'South Asia');
+              expect(element.data).to.have.length(1);
+            });
```
Comment (villebro) [inline superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts:113]: The legacy test is asserting the request, while the v1 test is asserting the response. Is it possible to check the v1 request here?

**Review by villebro — COMMENTED** [review]
One small comment, but looking mostly good 👍

File: superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts (hunk 1 above)
Comment (adam-stasiak) [reply superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts]: I found the way to access this data and fixed :) You can review this again @villebro

File: superset-frontend/cypress-base/cypress/integration/dashboard/url_params.test.js [hunk 2]
```diff
@@ -36,26 +37,26 @@ describe('Dashboard form data', () => {
...
-      });
+      return Promise.all(
+        requests.map(async xhr => {
+          expect(xhr.status).to.eq(200);
+          const responseBody = await readResponseBlob(xhr.response.body);
+
+          if (isLegacyResponse(responseBody)) {
+            const requestFormData = xhr.request.body;
+            const requestParams = JSON.parse(requestFormData.get('form_data'));
+            expect(requestParams).to.have.property('queryFields');
```
Comment (villebro) [inline superset-frontend/cypress-base/cypress/integration/dashboard/url_params.test.js:51]: This isn't being checked on v1 api. However, let's not add it, as we're in the process of removing it anyway.

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**